tests/components/evohome/snapshots/test_climate.ambr

```diff
@@ -230,21 +230,21 @@
         'permanent',
       ]),
       'status': dict({
-        'active_faults': list([
+        'active_faults': tuple(
           dict({
-            'faultType': 'TempZoneActuatorLowBattery',
-            'since': '2022-03-02T04:50:20',
+            'fault_type': 'TempZoneActuatorLowBattery',
```

Comment:
Let's also make sure this one stays consistent

Reply:
I hope this isn't going to stop **evohome** from achieving platinum tier!